game-core/src/main/java/games/strategy/engine/lobby/moderator/toolbox/tabs/AccessLogTab.java

```diff
@@ -0,0 +1,75 @@
+package games.strategy.engine.lobby.moderator.toolbox.tabs;
+
+import java.awt.Component;
+
+import javax.swing.JButton;
+import javax.swing.JTable;
+
+import org.triplea.swing.ButtonColumn;
+import org.triplea.swing.JButtonBuilder;
+import org.triplea.swing.JPanelBuilder;
+import org.triplea.swing.JTableBuilder;
+import org.triplea.swing.SwingComponents;
+
+import lombok.Builder;
+
+/**
+ * Show a scrollable list of 'banned users'.
+ * Bans are recorded by IP address and Hashed Mac.
+ *
+ * <pre>
+ * +--------------------------------------------------------------------------------+
+ * | +--------------------------------------------------------------------------+ |^|
+ * | | Name  | Access Date | IP  | Mac   | Registered |            |            | | |
+ * | +--------------------------------------------------------------------------+ | |
+ * | | name1 | Date        | IP1 |  mac1 |    T/F     | {Ban Name} | {Ban User} | | |
+ * | +--------------------------------------------------------------------------+ |v|
+ * +--------------------------------------------------------------------------------+
+ * </pre>
+ */
+@Builder
+final class AccessLogTab {
+  private final AccessLogTabActions accessLogTabActions;
+  private final AccessLogTabModel accessLogTabModel;
+
+
+  Component buildTab() {
+
+    final JButton loadMoreButton = JButtonBuilder.builder()
+        .title("Load More")
+        // .actionListener(() -> accessLogTabActions.loadMore(table))
```

Comment:
Not sure if this makes sense here, but we agreed that commented out code might as well be removed